nacl_test.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2018 ProximaX Limited. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package crypto
+
+import (
+	"testing"
+)
+
+func TestNacl_PackAndUnpackVerifyData(t *testing.T) {
```

Comment:
Is there going to be any tests?

Reply:
I think it isn't necessary because the tests in [crypto](https://github.com/proximax-storage/go-xpx-crypto/blob/b17c1685dd92e0b3c8b6a7f3e247b5c9fa908e21/crypto_test.go#L76) file ensure that the functionality we use from NaCl is working correctly both in producing the shared keys and the whole encryption/decryption process that relies on it.

In the future if we decide to expand that library or use other features we can add the tests. We can leave the empty file in there for the time being as a future notice if we do that.

How does that sound?

Reply:
Sounds good)